src/app/(BottomTabNavigation)/Cases/QRCodeScanner/AddCase/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useLocalSearchParams, useRouter } from 'expo-router';
+import React, { useEffect, useState } from 'react';
+import { Image, Text, TouchableOpacity, View } from 'react-native';
+import { getCase, uploadCase } from '../../../../../supabase/queries/case';
+import styles from './styles';
+
+enum caseStatus {
+  NotStarted,
+  InProgress,
+  Settled,
+  Cancelled,
+}
+
+export interface Case {
+  id: number;
+  approved: boolean;
+  title: string;
+  summary: string;
+  image: string;
+  caseStatus: caseStatus;
+}
+
+function AddCase() {
+  const local = useLocalSearchParams();
```

Comment:
Let's pass down the parameters id, title, summary, and image and give them types like this:
``` 
const { caseId, title, summary, image } = useLocalSearchParams<{
    caseId: CaseUid;
    title: string;
    summary: string;
    image: string;
  }>();
```